test/unit/BPool.t.sol

```diff
@@ -1006,18 +1006,28 @@ contract BPool_Unit_SwapExactAmountIn is BasePoolTest {
     bPool.swapExactAmountIn(tokenIn, _fuzz.tokenAmountIn, tokenOut, _minAmountOut, _maxPrice);
   }
 
-  function test_Revert_NotBoundTokenIn(address _tokenIn) public {
+  function test_Revert_NotBoundTokenIn(
+    address _tokenIn,
+    SwapExactAmountIn_FuzzScenario memory _fuzz,
+    uint256 _minAmountOut,
+    uint256 _maxPrice
+  ) public happyPath(_fuzz) {
     vm.assume(_tokenIn != VM_ADDRESS);
 
     vm.expectRevert('ERR_NOT_BOUND');
-    bPool.swapExactAmountIn(_tokenIn, 0, tokenOut, 0, 0);
+    bPool.swapExactAmountIn(_tokenIn, _fuzz.tokenAmountIn, tokenOut, _minAmountOut, _maxPrice);
```

Comment:
Could you fuzz all the function's general parameters within the `SwapExactAmountIn_FuzzScenario` struct?